### DISCUSSION TITLE
How to get values between 0 and 1 to display properly

### DISCUSSION THREAD

**Discussion by marinak** [Abhinandan-Kushwaha/react-native-gifted-charts#695] [General]
Thank you for the great plugin. 
My area chart is working fine apart from when I'm showing values that are low. Problems start at around 10 downwards and for my set of values [0.543, 0.544, 0.542, etc...] the line looks completely flat. Is there any way to fix this? Thanks much

![Screenshot 2024-07-04 at 13 50 00](https://github.com/Abhinandan-Kushwaha/react-native-gifted-charts/assets/73825518/7dbe704f-13e7-4860-9176-9031844d71df)

I would expect the chart to look something like this

<img width="283" alt="Screenshot 2024-07-04 at 14 08 14" src="https://github.com/Abhinandan-Kushwaha/react-native-gifted-charts/assets/73825518/a7b239cf-f4a2-4d46-8da5-7b2511129103">

**Comment by Abhinandan-Kushwaha** [comment]
Hi @marinak 
You can use the prop `stepValue` to set the value by which each step/section should increment on the y-axis. For your case, the `stepValue` should be `0.001`

Then you should use the prop `yAxisOffset = {0.539}`

You can also set the number of sections on the y-axis using the prop `noOfSections`. In your case it can be `noOfSections = {5}` 

I hope it helps.

**Reply by marinak** [reply]
That's great, thank you so much for your quick reply. 

One more related query - how can I allow the Y axis labels to be accurate as well, as at the moment I'm getting the same number for more than one steps?

![Screenshot 2024-07-05 at 10 19 12](https://github.com/Abhinandan-Kushwaha/react-native-gifted-charts/assets/73825518/eaa2b4ed-45f6-4bd7-b890-1afb752ca42e)

**Comment by Abhinandan-Kushwaha** [comment]
For accuracy of y-axis labels, you can use the `roundToDigits` prop.

Also, you can render custom text on y-axis using the `formatYLabel` prop. To know more about `formatYLabel` and `yAxisLabelTexts`, [see the docs here](https://gifted-charts.web.app/barchart?#yAxis)

**Reply by marinak** [reply]
Thanks again, much appreciated!
